packages/language-server/src/completion-items.ts

```diff
@@ -371,13 +371,34 @@ function rangeContains(range: Range, inner: Range): boolean {
   return atMost(range.start, inner.start) && atMost(inner.end, range.end);
 }
 
-function createInsertRange(line: number, column: number): Range {
+/**
+ * Create an insert range at the position right after the sent column.
+ * For exmaple, for text "12345" the character in position 1 is "1" and the
+ * insert position after it is "1|2345".
+ * The character in position 2 is "2" and the insert position after it is "12|345".
+ * The character at position 5 is "5" and the insert position after it at the end of the text -> "12345|".
+ * */
+function createInsertRangeAfter(line: number, column: number): Range {
   return {
     start: Position.create(line - 1, column),
```

Comment:
same as below, at minimum some comment about the magical `-1`

Reply:
I'll add a comment